catalog/spatial/geocoding/spatial-geocoding-create/src/main/java/org/codice/ddf/spatial/geocoding/create/GeoNamesCreator.java

```diff
@@ -33,6 +33,7 @@ public GeoEntry createGeoEntry(final String line) {
                 .featureCode(fields[7])
                 .population(Long.parseLong(fields[14]))
                 .alternateNames(fields[3])
+                .countryCode(fields[8])
```

Comment:
Actually, this `NumberFormatException` is already handled and rethrown as a `GeoEntryExtractionException`
https://github.com/codice/ddf/blob/master/catalog/spatial/geocoding/spatial-geocoding-extract/src/main/java/org/codice/ddf/spatial/geocoding/extract/GeoNamesFileExtractor.java#L123